tests/test_db.py

```diff
@@ -740,6 +768,14 @@ def test_identity_id_empty(self):
         with self.assertRaisesRegex(ValueError, IDENTITY_ID_EMPTY_ERROR):
             db.add_identity(uidentity, '', 'scm')
 
+    def test_identity_id_whitespaces(self):
+        """Check whether an identity with an ID composed by whitespaces cannot be added"""
+
+        uidentity = UniqueIdentity.objects.create(uuid=' ')
+
+        with self.assertRaisesRegex(ValueError, IDENTITY_ID_WHITESPACES_ERROR):
+            db.add_identity(uidentity, '\t  ', 'scm')
```

Comment:
This test is different to the other ones. You are comparing `\t    ' when others have only whitespaces. Why don't you add both checks to each test?

Reply:
You are right. Now three cases are tested: one with only whitespaces `'  '`, other with tab characters `'\t'` and other case mixing the other two: `'  \t  '`.